### ISSUE TITLE
Kubetest IAM Role

### ISSUE THREAD

**Issue by KrisJohnstone** [awslabs/amazon-eks-ami#1859]
Hey Guys,

In the docs you have a an IAM Policy [listed](https://github.com/awslabs/amazon-eks-ami/blob/main/doc/usage/overview.md), however this policy isn't suitable for the kubetest portion of the workflow. Wondering if you'd mind sharing it please :D?

**Comment by cartermckinnon** [comment]
Yep, that policy is just for building the AMI. I'll see if we can add an example policy to the kubetest2 deployer README.

Is your goal to re-use our CI workflows defined in this repo or just to use our kubetest2 deployers?

**Comment by KrisJohnstone** [comment]
> Yep, that policy is just for building the AMI. I'll see if we can add an example policy to the kubetest2 deployer README.
> 
> Is your goal to re-use our CI workflows defined in this repo or just to use our kubetest2 deployers?

Just the kubetest2 deployers :)
